content/guides/java/tracers.md

```diff
@@ -2,7 +2,67 @@
 title: "Java: Tracers"
 ---
 
-* Setting up your tracer
```

Comment:
How about a small intro here? Something like:

> A Tracer is the actual implementation that will record the spans and publish them somewhere. How your application handles the actual Tracer is up to you: either consume it directly throughout your application or store it in the GlobalTracer for easier usage with instrumented frameworks.

Reply:
Makes sense, will add it.